# This is the ROS equivalent to the PTScientists Rover Navigation Module
bool sequenceCount
bool mode
int16 xDistance
int16 yDistance
int16 rotationDistance
uint16 time